charts/rancher-vsphere-cpi/Chart.yaml

```diff
@@ -8,7 +8,7 @@ annotations:
   catalog.cattle.io/rancher-version: '>= 2.9.0-0'
   catalog.cattle.io/release-name: vsphere-cpi
 apiVersion: v1
-appVersion: 1.32.2
+appVersion: 1.32.3
```

Comment:
You didn't change the actual version of the `cloud-provider-vsphere` so it is not correct to bump this. Change only the chart version, below.